MVCForum/MVCForum.Website/Views/Members/LogOn.cshtml

```diff
@@ -48,7 +48,7 @@
                     </div>
 
                     <button type="submit" class="c-button c-button--min-width">
-                        @Html.LanguageString("LogOn.Button.Text")
+                        Log in
```

Comment:
As per previous discussions with Design/Fed Headers and Buttons are to be capitalized. Additionally I recall previous talk from design about log in looking similar to simple facebook design which is also capitalized. 

This change is conflicting with a previous update I believe, potentially could be dropped?